From a2c9eeb6064f4471d12b02f33e4309e894762731 Mon Sep 17 00:00:00 2001
From: Valentin Caron <valentin.caron@foss.st.com>
Date: Tue, 7 Jan 2025 11:05:08 +0100
Subject: [PATCH 1444/1695] dts: stm32: remove always-on from vdd33ucpd in
 stm32mp257f-ev1.dtsi

Remove always-on property from "vdd33ucpd" node and its parent "vdd3v3_usb"
On stm32mp257f-ev1 board.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git a6c06d3d2d06efa45a823ba34222375da40d8a94 dts: stm32: remove always-on from vdd33ucpd in stm32mp257f-ev1.dtsi]
Signed-off-by: Valentin Caron <valentin.caron@foss.st.com>
Change-Id: I76bf9dfcffc3ce80a8d007a2b40e3e2aabe715a7
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/427308
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Domain-Review: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
---
 core/arch/arm/dts/stm32mp257f-ev1.dts | 2 --
 1 file changed, 2 deletions(-)

diff --git a/core/arch/arm/dts/stm32mp257f-ev1.dts b/core/arch/arm/dts/stm32mp257f-ev1.dts
index cd997bf55..cc03c6d9d 100644
--- a/core/arch/arm/dts/stm32mp257f-ev1.dts
+++ b/core/arch/arm/dts/stm32mp257f-ev1.dts
@@ -269,7 +269,6 @@
 				regulator-name = "vdd3v3_usb";
 				regulator-min-microvolt = <3300000>;
 				regulator-max-microvolt = <3300000>;
-				regulator-always-on;
 				regulator-over-current-protection;
 			};
 			vpp_ddr: ldo5 {
@@ -365,7 +364,6 @@
 		regulator-min-microvolt = <3300000>;
 		regulator-max-microvolt = <3300000>;
 		vdd33ucpd-supply = <&vdd3v3_usb>;
-		regulator-always-on;
 	};
 
 	vdda18adc: vdda18adc {
-- 
2.47.2

